docs/content/preview/migrate/performance.md

```diff
@@ -24,39 +24,41 @@ There are several factors that slow down data-ingestion performance in any datab
 
 yb-voyager maximizes performance when migrating data into a newly created empty database in several ways:
 
-- Creates secondary indexes after the data import is complete. During the [import data](../migrate-steps/#import-data) phase, yb-voyager creates the secondary indexes after it completes data loading on all the tables. Then, it uses [Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md) to update indexes after data is loaded. This is much faster than online index maintenance.
+- Creates secondary indexes after the data import is complete. During the [post-import data](../migrate-steps/#import-indexes-and-triggers) phase, yb-voyager creates the secondary indexes after it completes data loading on all the tables except the unique indexes which are created in the [import-schema](../migrate-steps/#import-schema) phase, only to avoid any foreign key dependencies on the index. Then, it uses [Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md) to update indexes after data is loaded. This is much faster than online index maintenance.
```

Comment:
```suggestion
- Creates secondary indexes after the data import is complete. During the [post-import data](../migrate-steps/#import-indexes-and-triggers) phase, yb-voyager creates the secondary indexes (except the unique indexes) after it completes data loading on all the tables. Then, it uses [Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md) to update indexes after data is loaded. This is much faster than online index maintenance. (Unique indexes are created in the [import-schema](../migrate-steps/#import-schema) phase, to avoid any foreign key dependencies on the index.)
```